api/v1alpha1/osbuildenvconfig_types.go

```diff
@@ -146,6 +149,27 @@ type GenericS3ServiceConfig struct {
 	SkipSSLVerification *bool `json:"skipSSLVerification,omitempty"`
 }
 
+type ContainerRegistryServiceConfig struct {
```

Comment:
Please add that field to the sample yaml with setted values from the demo creation